recipe/meta.yaml

```diff
@@ -65,41 +65,41 @@ requirements:
     - posix                    # [win]
 
     # unknown
-    - {{native}}libjpeg-turbo  # [win]
-    - {{native}}libiconv       # [win]
-    - {{native}}gmp            # [win]
-    - {{native}}fftw           # [win]
-    - {{native}}mpfr           # [win]
-    - {{native}}libsndfile     # [win]
-    - {{native}}bwidget        # [win]
-    - {{native}}tktable        # [win]
+    - {{native}}libjpeg-turbo 1.4.*  # [win]
+    - {{native}}libiconv 1.14        # [win]
+    - {{native}}gmp 6.1.*            # [win]
+    - {{native}}fftw 3.3.*           # [win]
+    - {{native}}mpfr 3.1.*           # [win]
+    - {{native}}libsndfile 1.0.*     # [win]
+    - {{native}}bwidget 1.9.*        # [win]
+    - {{native}}tktable 2.10         # [win]
```

Comment:
Wouldn't the old versions still be avaialable? If these are dynamically linked libraries (which I don't know), we can indeed relax it again. Basically it was a try to react on your concern that statically linked libraries could impact the ABI of R when doing an update.